modules/microshift-install-bootc-build-image.adoc

```diff
@@ -28,7 +28,7 @@ ARG USHIFT_VER={product-version}
 RUN dnf config-manager \
         --set-enabled rhocp-${USHIFT_VER}-for-rhel-{op-system-version-major}-$(uname -m)-rpms \
         --set-enabled fast-datapath-for-rhel-{op-system-version-major}-$(uname -m)-rpms
-RUN dnf install -y firewalld microshift && \
+RUN install -y firewalld microshift container-tools <1>
```

Comment:
We do not install `container-tools` inside the image - only on the host that is used for building bootc images.

Reply:
This file should not be changed, IMO